data/Page/tips.md

```diff
@@ -0,0 +1,110 @@
+# How to look (and sound) good in your talk
+
+[This PDF by Kubecon EU is very useful](https://events.linuxfoundation.org/wp-content/uploads/2020/06/KubeCon_EU_Virtual_2020_WebPresenterDoc_v4.pdf), recommend reading.
+
+If you decide to buy additional hardware (note COVID is making stock levels challenging, order early), please make sure you put the time in to properly test before the event.  It’s unfortunately common to see someone buy an awesome new microphone to have them accidentally not have it set as the active recording audio device during their talk, or not have it placed in front of them correctly.
+
+## Microphone / audio quality
+
+By far, the best investment is ensuring audio quality is clear and not echo-y.  **This is more important than video quality.** Using your laptop microphone is OK for short talks or video chat, but the echo will get very fatiguing to listen to for a 20+ minute talk.
+
+In an ideal world; a external USB condenser microphone is your best bet, but you will need to practice with it to ensure you’re getting optimal audio levels, it’s positioned correctly, it doesn’t accidentally pickup your speakers/other sound during your talk, etc.
```

Comment:
s/etc/and make sure any gain or volume controls are set so it's loud and clear, without any distortion, even when you're speaking loudly into it.